assets/js/modules/analytics-4/components/dashboard/ActivationBanner/ReminderBanner.js

```diff
@@ -87,6 +95,28 @@ export default function ReminderBanner( {
 		ACTIVATION_ACKNOWLEDGEMENT_TOOLTIP_STATE_KEY
 	);
 
+	const viewContext = useViewContext();
+	const eventCategory = `${ viewContext }_ga4-reminder-notification`;
+
+	useMount( () => {
+		trackEvent( eventCategory, 'view_notification' );
+	} );
+
+	const handleCTAClick = useCallback( async () => {
+		await trackEvent( eventCategory, 'confirm_notification' );
+		return onSubmitSuccess();
+	}, [ eventCategory, onSubmitSuccess ] );
+
+	const handleDismiss = useCallback( async () => {
+		await trackEvent( eventCategory, 'dismiss_notification' );
+		showTooltip();
+	}, [ eventCategory, showTooltip ] );
+
+	const handleLearnMore = useCallback(
+		() => trackEvent( eventCategory, 'click_learn_more_link' ),
+		[ eventCategory ]
+	);
```

Comment:
Nitpick, but 

```suggestion
	const handleLearnMore = useCallback( () => {
		trackEvent( eventCategory, 'click_learn_more_link' );
	}, [ eventCategory ] );
```

looks more like the other callbacks here and seems to make more sense, since we aren't actually utilising the implicit return here. This seems to be done for terseness, but it's no fewer lines and I think is actually a bit less clear, because the reader needs to sort out that the return value doesn't actually matter here. 😅

Reply:
@tofumatt True, the idea is that it should only trigger when the notification is actually shown. Good catch, so I guess the two conditions here `isTooltipVisible` and `isLoadingAnalyticsAccess && ! isDismissed` that do not render the `BannerNotification` should, strictly speaking, not trigger it.